tests_e2e/playwright/.env.tpl

```diff
@@ -0,0 +1,4 @@
+USERNAME=""
+PASSWORD=""
+NEBARI_FULL_URL=""
+INSTANCE_NAME=""
```

Comment:
I will add some info on what goes in here. I would say that the tests should be small enough to use the (default) small instance type. The Navigator class can handle alternative instance types... but in order to leverage that we would have to shift to creating Navigator instances (setup and login) for every test. This is because we can't have two playwright objects running at the same time.

Reply:
Fair enough. Should we actually hard-code the instance type then? Not sure if we need to collect it from an environment variable in that case.